Explorer/Assets/DCL/Character/CharacterMotion/Systems/InterpolateCharacterSystem.cs

```diff
@@ -38,6 +39,9 @@ protected override void Update(float t)
         [Query]
         private void TeleportPlayer(in Entity entity, in CharacterController controller, ref CharacterPlatformComponent platformComponent, in PlayerTeleportIntent teleportIntent)
         {
+            if (teleportIntent.LoadReport != null && teleportIntent.LoadReport.CompletionSource.UnsafeGetStatus() == UniTaskStatus.Pending)
```

Comment:
Why is it checking by LoadReport? It's not marked as nullable

Reply:
LoadReport marked as nullable: https://github.com/decentraland/unity-explorer/pull/717/commits/76fce5394aa192c52e6e656a8ef7676df11a4437

Reply:
Done: https://github.com/decentraland/unity-explorer/pull/717/commits/2ce7a9903a454fa406ae323a1dc6129ced10eb3e